src/Logo/Logo.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LogoColors, LogoTypes } from './Logo.levers';
+
+/**
+ * Stanford Wordmark Logo Component.
+ *
+ */
+export const Logo = ({ className, ...props }) => {
+  const classnames = require('classnames');
+  const levers = {};
+  let logoText;
+
+  // Levers
+  // ---------------------------------------------------------------------------
+
+  // props.color
+  if (props.color && LogoColors.includes(props.color)) {
+    switch (props.color) {
+      case 'cardinal-red':
+        levers.logo = classnames('su-text-cardinal-red');
+        break;
+
+      case 'black':
+        levers.logo = classnames('su-text-black hover:su-text-black focus:su-text-black');
+        break;
+
+      case 'white':
+        levers.logo = classnames('su-text-white hover:su-text-white focus:su-text-white');
+        break;
+    }
+  }
+
+  // props.type
+  if (props.type && LogoTypes.includes(props.type)) {
+    switch (props.type) {
+      case 'short':
+        logoText = 'Stanford';
+        break;
+
+      case 'full':
+        logoText = 'Stanford University';
+        break;
+
+      case 'stacked':
+        logoText = 'Stanford \n University';
```

Comment:
@sherakama You might have a quick answer to this - what's the best way for me to pass in "Stanford<br />University" as HTML here?

Reply:
Perhaps something like this?

```suggestion
        logoText = (<>Stanford <br /> University</>);
```

Reply:
Yay! That works - thank you!